_releases/2023-01-09-december-2022.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+category: releases
+title: December 2022 Release Notes
+filename-custom: december-2022
+last_modified_at: January 9, 2023
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements 
+* **ElasticSearch Upgrade**: We've completed the upgrade to our ElasticSearch version across our system.
+* **Rails Upgrade**: We've completed the upgrade to our Rails version in our main application.
+* **SERP Redesign**: We've installed the necessary components to support React in our Rails application.
+* **Indexing Improvements**: We've added the ability for any member of our team to queue a specific set of URLs for reindexing, rather than the entire domain. 
```

Comment:
how about "to enqueue for reindexing a specific set of URLs, rather than the entire domain"

Reply:
updated!